### PR TITLE
Feature/speedup

### PR DESCRIPTION
**Pull request by tgurbich**
- `--fast` flag implementation
- improvement of eggNOG function description that is added to the final GFF

### CODE REVIEW

File: modules/local/eggnog.nf [hunk 1]
```diff
@@ -44,6 +42,8 @@ process EGGNOG_MAPPER {
         --data_dir ${eggnog_db_dir} \
         --no_file_comments \
         --cpu ${task.cpus} \
+        --tax_scope 'prokaryota_broad' \
```
Comment (mberacochea) [inline modules/local/eggnog.nf:45]: Should we consider the lineage here?

**Review by mberacochea — APPROVED** [review]
Good stuff! I left one question in EggNOG, but other than that it looks good to me

File: modules/local/eggnog.nf (hunk 1 above)
Comment (tgurbich) [reply modules/local/eggnog.nf:45]: That's very tricky to do on diverse taxa some of which might be unknown. I think having prokaryota_broad as a scope is a good compromise overall - better than no tax scope but shouldn't create issues for unknown taxonomies.

Comment (mberacochea) [reply modules/local/eggnog.nf:45]: and `archea`?

**Pull request merged**